tempodb/config.go

```diff
@@ -60,8 +63,33 @@ type Config struct {
 }
 
 type SearchConfig struct {
+	// v2 blocks
 	ChunkSizeBytes     uint32 `yaml:"chunk_size_bytes"`
 	PrefetchTraceCount int    `yaml:"prefetch_trace_count"`
+
+	// vParquet blocks
+	ReadBufferCount int `yaml:"read_buffer_count"`
+	ReadBufferSize  int `yaml:"read_buffer_size"`
```

Comment:
One other option is to have a `Config` in `encoding/vparquet` and embed that struct into the `SearchConfig` here, kind of like - 

```

type SearchConfig struct {
	ChunkSizeBytes     uint32 `yaml:"chunk_size_bytes"`

	ParquetConfig      vparquet.SearchConfig    `yaml:","`
}
```

Reply:
Might consider adjusting the field names.  I think our convention is to include `...Bytes` for clarity, like `ChunkSizeBytes`.

Reply:
That's not a bad idea but we'd need to figure out a way to have that roll forward with future schemas.